### ISSUE TITLE
Improve scoring

### ISSUE THREAD

**Issue by derekfountain** [derekfountain/zxwonkyonekey#11]
There has to be a reason for playing and a score is surely it! I need to add a time based score.

I've also considered a slow down key, to stop it being frustratingly fast. Perhaps a bonus is in order, which is depleted when the slowdown is in use?

**Comment by derekfountain** [comment]
The 2 levels I have so far both happen to have clear areas in their bottom right corners where a score and bonus or timer sliders could be placed. Not sure what's required yet.

**Comment by derekfountain** [comment]
I've added a score and a bonus. The score values are way too high, and the bonus doesn't do anything yet. It's a start though. :)

**Comment by derekfountain** [comment]
The scoring is dull. Just numbers, and they don't mean anything. Spectrum games have traditionally had more imaginative scoring systems. I need to come up with something for score and bonus.

**Comment by derekfountain** [comment]
I think I'm going to change the bonus to a number - say 3 or 5 - sprites which, when a slowdown is consumed, fall of the shelf and disappear. That would add a bit more dynamism and animation.

I'd like to do the scoring in a similar way, but I'm not quite sure yet.

**Comment by derekfountain** [comment]
Another idea: use both lines of the screen to animate a slug crawling towards its den. When it gets there, change it to say "no points!". Taking slowdowns speeds the slug up.

**Comment by derekfountain** [comment]
I changed the bonuses to 5 apples, each one lost on consumption of a slowdown pill. I wanted the apple to sort of fall off its shelf when consumed, and wrote the code to do it, but it slowed the game below 50fps. I'm right on the edge of 50fps performance. So I changed it so the apple just disappears, which isn't as satisfying but at least it's a tolerable compromise.

I still need to do the score. I really want to get that utoa() call out of the main loop...
